### DISCUSSION TITLE
I made a destiny 2 aimbot using yolov5!

### DISCUSSION THREAD

**Discussion by rlewkowicz** [ultralytics/yolov5#10651] [Show and tell]
The code is hot trash, needs a full rewrite. Model it's self is not public and god help you trying to compile the code lol.

Still a cool deal:
https://hireryan.today/2023/01/02/just-a-better-gaming-chair-honestly-destiny-2-aim-bot-using-yolov5/

**Comment by meowsynth** [comment]
oh!

**Comment by meowsynth** [comment]
can i add you in discord?
